macros/core/PGbasicmacros.pl

```diff
@@ -3023,7 +3040,14 @@ sub image {
 	while (@image_list) {
 		my $image_item = shift @image_list;
 		if (ref $image_item eq 'parser::GraphTool') {
-			push(@output_list, $image_item->generateAnswerGraph(ariaDescription => shift @alt_list // ''));
+			push(
+				@output_list,
+				$image_item->generateAnswerGraph(
+					width           => $width,
+					height          => $height,
+					ariaDescription => shift @alt_list // ''
```

Comment:
Here is one way to do what I am suggesting.
```suggestion
				$image_item->generateAnswerGraph(
					$out_options{width} || $out_options{height} ? (width => $width, height => $height) : (),
					ariaDescription => shift @alt_list // ''
```